<img src="https://coursesandconferences.wellcomeconnectingscience.org/wp-content/themes/wcc_courses_and_conferences/dist/assets/svg/logo.svg" width="300" height="50"> 

# Single Cell Genomic Approaches to Study the Immune System - Latin America & the Caribbean

## 9–11 November 2024, Universidad CES, Medellín, Colombia

[Wellcome Connecting Science Course Run Website Link](https://coursesandconferences.wellcomeconnectingscience.org/event/single-cell-genomic-approaches-to-study-the-immune-system-latin-america-the-caribbean-20241109/) <br /> 
[Course Time Table](https://docs.google.com/viewer?url=https://raw.githubusercontent.com/WCSCourses/Single_Cell_Genomics_ImmuneSys_24/main/Single_Cell_JAGUAR_Course_Time_Table.pdf) <br /> 
[Course Informatics Guide](Coming Soon..)

# Summary

In collaboration with [Project Jaguar](https://www.sanger.ac.uk/collaboration/project-jaguar/), [ICMT Universidad CES](https://www.ces.edu.co/) and [Universidad de Antioquia](https://www.udea.edu.co/wps/portal/udea/web/inicio/!ut/p/z1/04_Sj9CPykssy0xPLMnMz0vMAfIjo8zi_QJNXQ2NnA18_D2NXQ0CLf1MA4zdPY1MzI31wwkpiAJKG-AAjgZA_VFgJRaWzkaGjiYGPgbehqYGjoGuAX4h_s4BLl5GUAV4zCjIjTDIdFRUBAC3KPbO/dz/d5/L2dBISEvZ0FBIS9nQSEh/), we are excited to offer a complimentary workshop on applying single-cell genomic techniques to investigate the immune system. Single-cell immunogenomics approaches help to understand how environmental factors and genetic backgrounds influence immune responses. This course is designed to equip scientists with the skills and resources to obtain and analyse single-cell omics data. We will highlight the interplay between experimental design, laboratory processing, and downstream analysis, sharing insights from our experiences in profiling the immune system and conducting in vitro stimulations.

In this course we will cover a toolkit of single-cell technologies, including scRNAseq, CITEseq, ATACseq, and immune receptor repertoires in single cell, discussing their benefits, limitations, and best practices. Additionally, we will share our experiences from a large-scale single-cell immunogenomics project involving researchers from various institutions based in seven Latin American countries working in partnership with Wellcome Sanger Institute (Project JAGUAR).

This course will be held preceding the [ISCB Latin America Conference on Bioinformatics](https://www.iscb.org/latam2024/home).

# Target audience: 

Early and mid-career biomedical scientists, MSc, PhD students and postdocs, who are beginning or planning to begin single-cell omics research in the field of immunology, are invited to participate. This course is open to applicants based in Latin America.

A basic proficiency in R or Python is required. Pre-course materials on R and Linux will be provided to prepare participants for the course.

## Learning outcomes

**What will you be able to do?**

After attending this course, participants will be able to:

- Identify critical decisions for the data interpretability in their experimental planning and design with a focus on the immune system.
- Decide the appropriate steps for dataset preprocessing and analysis and apply them within the framework for immunological interpretation.
- Operate both R and Python-based analytical frameworks to conduct single-cell analysis.
- Understand the integration of samples and omics datasets e.g. transcriptomics and CITE-seq, across single cell experiments and how to leverage and interpret the results.
- Describe downstream analysis types appropriate for their biological questions in studying the immune system.

# Programme

This hands-on bioinformatics workshop will offer a series of lectures and practical sessions covering the following topics:

- Single-cell basics and immune system applications: From the technology to the implementations.
- Best practices for experimental planning and design focusing on the study of the immune system.
- Single-cell RNA sequencing: from raw sequences to count matrix and downstream analysis through comprehensive data workflows.
- Immune receptor repertoires in single-cell technologies.
- CITE-seq technology to profile cells at transcriptomics and proteomic levels.
- ATAC-seq and multiome analysis.

**Scientific Organising Committee**      

- [Yesid Cuesta Astroz](https://scholar.google.es/citations?user=q0hiAjEAAAAJ&hl=es), University of Antioquía, Colombia
- [Gosia Trynka](https://www.sanger.ac.uk/person/trynka-gosia/), Wellcome Sanger Institute, UK
  
**Course Instructors**      

- [Anna (Ania) Lorenc](https://www.sanger.ac.uk/person/lorenc-anna/), Wellcome Sanger Institute, UK
- [Benilton Carvalho](https://portal.dados.unicamp.br/perfil?origem=&docente=309039&sigla_unidade=IMECC&nome_unidade=INSTITUTO%20DE%20MATEMATICA,%20ESTATISTICA%20E%20COMPUTACAO%20CIENTIFICA&nome_programa=), UNICAMP, Brazil 
- [Carla Jones](https://www.sanger.ac.uk/person/jones-bell-carla/), Wellcome Sanger Institute, UK
- [Danilo Ceschin](https://www.iucbc.edu.ar/cimetsa/investigacion-traslacional-en-epigenetica-y-aterosclerosis.html), CIMETSA - IUCBC, Argentina
- [Diego Ramirez](), UNAM, Mexico
- [Evelia Coss](https://eveliacoss.github.io/), UNAM, Mexico
- [Felipe Gajardo](https://www.researchgate.net/profile/Felipe-Gajardo-3), PUC, Chile 
- [Julieth Lopez](https://co.linkedin.com/in/julieth-andrea-lopez), UNAL, Colombia 
- [Lucia Ramirez](https://scholar.google.com/citations?user=nESolKwAAAAJ&hl=es), Wellcome Sanger Institute, UK

**Wellcome Connecting Science Team**   

- [Alice Matimba](https://uk.linkedin.com/in/alice-matimba-8805177), Head of Training and Global Capacity
- [Isabela Malta](https://www.linkedin.com/in/isabela-malta-754503162/), Assistant Global Training Manager
- [Karon Chappell](https://coursesandconferences.wellcomeconnectingscience.org/about-us/the-team/), Event Organiser
- [Jorge Batista da Rocha](https://www.wellcomeconnectingscience.org/person/batista-da-rocha-jorge/), Laboratory Operations Officer
- [Martin Asltett](https://coursesandconferences.wellcomeconnectingscience.org/about-us/the-team/), Education Developer
- [Vaishnavi Vikas Gangadhar](https://www.wellcomeconnectingscience.org/person/gangadhar-vaishnavi/), Informatics Technical Officer

******

# Bioinformatics Material

### Lectures

These lectures aim to complement practical modules focused on immune system applications. They will cover the fundamentals of single-cell technology, guide you through designing an experiment, and explore how these technologies are used to study immune system regulation.

Date: 9-11 November

- Single cell basics and immune system applications. Basics of technology and their implementations. From raw sequences to counting matrix [Slides](https://docs.google.com/viewer?url=https://github.com/WCSCourses/Single_Cell_Genomics_ImmuneSys_24/raw/main/modules/Module1_scRNAseq/Module1_intro_scRNAseq.pdf)
- How to design a scRNaseq experiment [Slides](https://docs.google.com/viewer?url=https://github.com/WCSCourses/Single_Cell_Genomics_ImmuneSys_24/raw/main/modules/Lectures/single_cell_course_Exp_design_Medellin_2024.pdf)
- Unprecedented insights into immune system regulation with single-cell technologies [Slides]()

### Module 1: scRNA-seq tutorial

Date: Saturday 9 November

Single-cell RNA sequencing (scRNA-seq) technology revolutionizes our understanding of cellular heterogeneity by enabling transcriptomic analysis at the single-cell level. This tutorial will explain the basis of single-cell and immune system applications through a lecture that delves into the principles and methodologies of scRNA-seq, with focus on how they shape and influence analysis. Key technological implementations will be discussed. More technical part of this talk will outline processing the sequencing data, from raw reads in FASTQ files to preprocessing-ready count matrices and methods for cleaning and adjusting data in preprocessing (identification of bad quality cells and multiplets, demultiplexing pooled samples). Finally, this tutorial will offer a computational practice using open data to understand the workflow of scRNA-seq data analysis.

- Presentations:
  - Intro to scRNAseq [Slides](https://github.com/WCSCourses/Single_Cell_Genomics_ImmuneSys_24/blob/main/Module1_intro_scRNAseq.pdf)
  - Data structure and basic quality control [Slides](https://docs.google.com/presentation/d/1Tr5-qyDVdFsrk9WQzGv3gNhx6OHQk7P-35dd8ODXKF0/edit?usp=sharing)
  - Data normalization and clustering [Slides](https://benilton.github.io/wcs-rio/norm_clust.html)
  - Differential expression and cell type annotation [Slides](https://benilton.github.io/wcs-rio/de_annot.html)
- Tutorials:
  - Practical 1 to 4: scRNAseq tutorial [Google Colab Notebook](https://colab.research.google.com/github/WCSCourses/Single_Cell_Genomics_ImmuneSys_24/blob/main/modules/Module1_scRNAseq/scRNAseq.ipynb)
  - scRNAseq workshop solutions [Google Colab Notebook](https://colab.research.google.com/github/WCSCourses/Single_Cell_Genomics_ImmuneSys_24/blob/main/modules/Module1_scRNAseq/scRNAseq_Workshop_Solutions.ipynb)

### Module 2: CITE-seq tutorial

Date: Sunday 10 November

CITE-Seq (Cellular Indexing of Transcriptomes and Epitopes by sequencing) is an advanced technique that merges single-cell RNA sequencing (scRNA-seq) with protein marker detection, enabling comprehensive profiling of individual cells at both the transcriptomic and proteomic levels. By attaching unique oligonucleotide barcodes to antibodies that bind specific cell surface proteins, CITE-Seq allows for simultaneous measurement of gene expression and protein abundance in thousands of single cells. This dual-modality approach offers deeper insights into cellular heterogeneity, states, and functions compared to traditional methods. CITE-Seq has proven invaluable in various fields, including immunology, oncology, and developmental biology, providing a powerful tool for deciphering complex biological systems and advancing precision medicine. This technique's ability to integrate RNA and protein data from the same cells presents new opportunities for biomarker discovery, disease characterization, and therapeutic development. This tutorial will explain the basis and the computational analysis of CITE-seq data.

- Presentations:
  - CITEseq tutorial [Slides](https://drive.google.com/drive/folders/1sdqfONM6nBs1M3jt_HO1-Qy2PWJRsPaW?usp=sharing)
- Tutorials:
  - Practical 5: The basic structure of a multimodal Seurat Object [Google Colab Notebook](https://colab.research.google.com/drive/18LiunIhZN-2VqrPzJz4tSlJ-yV8KuFNE?usp=drive_link)
  - Practical 6: Background correction and normalization [Google Colab Notebook](https://colab.research.google.com/drive/17kOUk4lqE_J52nrozPv7pEK5x-u6XR5W?usp=sharing)
  - Practical 7: Visualization of protein  levels and cell populations [Google Colab Notebook](https://colab.research.google.com/drive/1gTNSD403yT15_5HRwW5bj91R7-5tmJTB?usp=sharing)
  - Practical 8: Options for CITE-seq-based annotation [Google Colab Notebook](https://colab.research.google.com/drive/1Gcv4bPaNCA9uUF7qdf3NvW7UjdCgaQDd?usp=sharing)
  - Practical 9: Integration of multiple batches [Google Colab Notebook](https://colab.research.google.com/drive/1jwHP76SbX6JQNjZwLgY_fEje_Lky39OL?usp=drive_link)

### Module 3: Between groups analysis and Immune receptors

Date: Sunday 10 November 

Single cell technologies provide significant opportunities for biological discovery, but the laboratory methods and intrinsic sample variability pose challenges when analyzing samples together. Similarly, the sparse nature of expression data requires special statistical handling when comparing samples across groups. This module present ways to deal with batch effects and variance-reducing methods in downstream analyses of scRNAseq. On the other hand, this module cover the basics of receptor biology and explain how receptor sequences are obtained, both through targeted methods and serendipitous discovery. It will address the challenges involved in interpreting these sequences and explore the various applications of this data. In the practical section, participants will work with single-cell RNA sequencing (scRNA-seq) data that contains receptor information. The session will include an introduction to accessing this data and outline the typical benefits it provides.

- Presentations:
  - Lecture: Combining samples together in the analysis to answer biological questions [Slides](https://docs.google.com/presentation/d/1Z4OODsGVHO0xAw0wfMbCs6_sI_H6rKs301Y0KiTeUdA/edit?usp=sharing)
  - Immune receptor repertoires in single-cell technologies [Slides](https://docs.google.com/presentation/d/1ewVDWzZE07QZlRBSYCPLZ4g6hEeLtJFz/edit?usp=sharing&ouid=104245724826540258577&rtpof=true&sd=true)
- Tutorials:
  - Practical 11: Differential expression analysis [Google Colab Notebook](https://colab.research.google.com/github/WCSCourses/Single_Cell_Genomics_ImmuneSys_24/blob/main/modules/Module3_MultiSample_ImmuneReceptor_Analysis/Module_3_%20dge.ipynb)
  - Practical 12: Immune receptor repertoires in single-cell technologies [Google Colab Notebook](https://colab.research.google.com/github/WCSCourses/Single_Cell_Genomics_ImmuneSys_24/blob/main/modules/Module3_MultiSample_ImmuneReceptor_Analysis/Module_3_Immune_receptor_repertoires.ipynb)

### Module 4: ATAC-seq tutorial

Date: Monday 11 November

Single-cell transposase-accessible chromatin sequencing (scATAC-seq) represents the most innovative technology for examining genome-wide regulatory landscapes in single cells. For this tutorial, we will be analyzing a single-cell ATAC-seq dataset of human peripheral blood mononuclear cells (PBMCs) provided by 10x Genomics. We will run bridge integration for PBMC with the newly released Azimuth ATAC workflow. In this workshop we will review the existing statistical tools for analyzing scATAC-seq data, how to document your analysis and review some tools for interpreting results.

- ATACseq tutorial [Slides]()
- ATACseq tutorial [Google Colab Notebook]()

******

## Citing and Re-using Course Material

The course data are free to reuse and adapt with appropriate attribution. All course data in these repositories are licensed under the <a rel="license" href="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</a>. <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons Licence" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-sa/4.0/88x31.png" /></a><br /> 

## Interested in attending a course?

Take a look what courses are coming up at [Wellcome Connecting Science Courses & Conference Website](https://coursesandconferences.wellcomeconnectingscience.org/our-events/).

---

For more information or queries, feel free to contact us via the [Wellcome Connecting Science website](https://coursesandconferences.wellcomeconnectingscience.org).<br /> 


Find us on socials [Wellcome Connecting Science Linktr](https://linktr.ee/eventswcs)

---

[Wellcome Connecting Science GitHub Home Page](https://github.com/WCSCourses) <br /> 
